pkgs/by-name/go/golem/package.nix

```
@@ -0,0 +1,55 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  pkg-config,
+  openssl,
+  protobuf,
+  redis,
+  fontconfig,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "golem";
+  version = "1.0.23";
+
+  src = fetchFromGitHub {
+    owner = "golemcloud";
+    repo = pname;
+    rev = "v${version}";
```

```suggestion
    rev = "refs/tags/v${version}";
```
To prevent a branch/tag mismatch.